css/amp-story-player.css

```diff
@@ -33,6 +33,12 @@ amp-story-player a:not(:first-of-type) {
   visibility: hidden;
 }
 
+amp-story-player.i-amphtml-story-player-loaded a:first-of-type {
```

Comment:
Can you explain what this does and why it is needed in this PR? If you're trying to hide the links, should we just `display: none;`? Also, why is this not an issue when Shadow DOM is supported?

Reply:
Why is this an issue, don't the stories iframe cover it?

Reply:
Nope, when they actually get a size the iframe gets pushed below the `<a>` tag

Reply:
As discussed offline, let's make sure the iframe container covers whatever is inside the <amp-story-player> tag by using a position absolute :))

Reply:
As discussed offline: when swiping in the last story, the `<a>` tag and whatever is inside of it will be visible. We think it's better to just hide it.

Reply:
Why are we only hiding the first one though? Shouldn't we hide all the content within the amp-story-player tag, even if publishers add an <img> or a <p> or multiple <a>?

Reply:
Yeah, for now we only hide the first one because the player explicitly sets styles on it to display the poster image while the story loads. But we can't know for sure what publisher does inside of it. Maybe we could do a 
`amp-story-player > *{ display:none } amp-story-player > .main-container { display:block }`. WDYT?

Reply:
`amp-story-player.loaded > *{ display:none }` yes :)) 
You'd need to test it with and without shadow DOM.
You have more context but from what I know it sounds safer?